hasjob/templates/tablayout.html.jinja2

```diff
@@ -31,4 +32,5 @@
   <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3.min.js"></script>
   <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/c3/0.4.14/c3.min.js"></script>
```

Comment:
@jace should these static files be in tablayout template? Shouldn't they be in individual templates that inherit them and needs c3/d3?